gpt4all-chat/download.cpp

```diff
@@ -199,20 +207,27 @@ void Download::removeModel(const QString &modelFile)
         incompleteFile.remove();
     }
 
+    bool shouldRemoveInstalled = false;
     QFile file(filePath);
     if (file.exists()) {
         const ModelInfo info = ModelList::globalInstance()->modelInfoByFilename(modelFile);
-        ModelList::globalInstance()->removeInstalled(info);
+        shouldRemoveInstalled = info.installed && !info.isClone() && (info.isDiscovered() || info.description() == "");
```

Comment:
Could we keep at least the "indicates sideloaded" comment from removeInstalled?

Reply:
yes

Reply:
fixed